python/src/scipp/compat/mantid.py

```diff
@@ -2,15 +2,15 @@
 # Copyright (c) 2019 Scipp contributors (https://github.com/scipp)
 # @author Simon Heybrock, Neil Vaytet
 
-import scipp as sc
+from .._scipp import core as sc
```

Comment:
Why this change `.._scipp` implies we are now intending to use it from CMAKE_BINARY directory directly?

Reply:
Previously `scipp` imported `neutron`, which imported `load` which imported `scipp`, so there was a cyclic dependency. Apparently Python managed to deal with it, but it did not seem like a good setup. It also appeared to cause some issues when trying to import submodules of `scipp` directly.

I don't know if this is the best solution, let me know if there is a better way.

Reply:
Ok. Makes sense, and I don't have a better solution for handling that.